content/docs/0.8.1/snapshots-and-backups/setup-disaster-recovery-volumes.md

```diff
@@ -8,16 +8,48 @@ A **disaster recovery (DR) volume** is a special volume that is mainly intended
+## Creating DR Volumes
 
-## Creating DR Volumes {#creating}
+> **Prerequisites:** The steps to create a DR volume assume that you have done the following:
+>
+>    1. You have set up two Kubernetes clusters. These will be called Cluster A and Cluster B.
+>    2. Longhorn is installed on both clusters.
+>    3. The same [backup target has been configured](../backup-and-restore/set-backup-target) for Longhorn on both clusters.
+>    4. You have [created a volume](../../volumes-and-nodes/create-volumes) on Cluster A. This will be called Volume X.
+>    5. You have [created a backup](../backup-and-restore/create-a-backup) of this volume. This backup will be used to a create a DR volume on Cluster B.
 
-> **Prerequisites:** Set up two Kubernetes clusters. These will be called cluster A and cluster B. Install Longhorn on both clusters, and set the same backup target on both clusters. For help setting the backup target, refer to [this page.](../backup-and-restore/set-backup-target)
+To create a DR volume from Volume X, follow these steps:
 
-1. In the cluster A, make sure the original volume X has a backup created or has recurring backups scheduled.
-2. In backup page of cluster B, choose the backup volume X, then create disaster recovery volume Y. It's highly recommended to use the backup volume name as the disaster volume name.
-3. Longhorn will automatically attach the DR volume Y to a random node. Then Longhorn will start polling for the last backup of volume X, and incrementally restore it to the volume Y.
+1. In the Longhorn UI for Cluster B, click the **Backup** tab.
+2. Select the table row for Volume X, then click **Create Disaster Recovery Volume.** It's highly recommended to use the backup volume name as the DR volume name.
+3. Longhorn will automatically attach the DR volume Y to a random node.
 
-> **Note:** Only DR volumes can be deactivated. Deactivating a normal volume is not allowed.
+**Result:** The DR volume is created. Longhorn will start polling for the last backup of Volume X, and incrementally restore it to the DR volume.
+
+## Activating DR Volumes
+
+Because the main purpose of a DR volume is to restore data from backup, this type of volume doesn't support the following actions before it is activated:
+
+- Creating, deleting, and reverting snapshots
+- Creating backups
+- Creating persistent volumes
+- Creating persistent volume claims
+
+To activate the DR volume,
+
+1. Go to the Longhorn UI in your backup cluster where the DR volume is located. Click the **Volume** tab.
+2. Go to the DR volume. In the **Operations** column, click the three-line dropdown menu, and click **Activate Disaster Recovery Volume.**
+3. Click **OK.**
+
+When a DR volume is activated, the following happens:
+
+1. Longhorn will check the last backup of the original volume.
+2. If the backup hasn't been restored, the restore will be started, and the activate action will fail. You will need to wait for the restore to complete before retrying.
```

Comment:
As we discussed, this is no longer true.

Reply:
Replaced lines 46 to 49 with:

> **Result:** Longhorn will make sure the DR volume is updated to reflect the most recent backup of its original backup volume, then activates the volume.